schema/testing.proto

```diff
@@ -238,4 +230,128 @@ message Test {
 
   // A description of the test.
   string description = 3;
+
+  // A blueprint is an explicit definition of methods and requests that are needed
+  // to be made to test this specific test case. Ideally this would be represented
+  // by something more robust like CEL, but as of writing this, I am unsure if CEL
+  // is ready.
+  message Blueprint {
+    // The id of this blueprint.
+    string id = 1;
+
+    // A description of this blueprint.
+    string description = 2;
+
+    // A message representing a method invocation.
+    message Invocation {
+      // The fully qualified name of the showcase method to be invoked.
+      string method = 1;
+
+      // The request to be made if a specific request is necessary.
+      google.protobuf.Any request = 2;
```

Comment:
The `type_url` is not going to do anyone any good here; this could just be a bytes:

```proto
bytes serialized_request = 2;
```

Reply:
Ah agreed. Changed.